dev/src/bundle.ts

```diff
@@ -49,10 +46,7 @@ export function validateDocumentSnapshot(
  * @param arg The argument name or argument index (for varargs methods).
  * @param value The input to validate.
  */
-export function validateQuerySnapshot(
-  arg: string | number,
-  value: unknown
-): void {
+function validateQuerySnapshot(arg: string | number, value: unknown): void {
```

Comment:
Please do move to bottom of the module to match the code layout in the rest of the repo.